backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -330,7 +330,10 @@ public void setOnRequestHandler(RequestHandler requestHandler) {
      * @return true if we successfully enabled breadcrumbs
      */
     public boolean enableBreadcrumbs(Context context) {
-        return database.getBreadcrumbs().enableBreadcrumbs(context);
+        if (isBreadcrumbsEnabled()) {
```

Comment:
do i understand it corretly? if breadcrumbs are enabled, enable breadcrumbs otherwise return false? It sounds weird. Something is odd

Reply:
now after refactor this method is called isBreadcrumbsAvailable so breadcrumbs can be enabled only when it's available